events/handler.js

```diff
@@ -1,14 +1,10 @@
 'use strict';
 
-const AWS = require('aws-sdk')
-const uuid = require('uuid')
+import Processor from './processor';
 
-module.exports.errors = (event, context, callback) => {
-  event.Records.forEach((record) => {
-    var buf = new Buffer(record.kinesis.data, 'base64').toString('ascii');
-    var data = JSON.parse(buf); 
-    console.log("Log data received = " + JSON.stringify(data))
+module.exports.process_event = (event, context, callback) => {
```

Comment:
I would say if you are gonna be doing javascript might as well lean into it and go `camelCase` otherwise your own methods are gonna be formatted differently than any library you use